src/action/index.js

```diff
@@ -0,0 +1,7 @@
+export function updateTitle({data, type}){
+    return { type, data }
+};
+
+export function handleShowModal({data, type}) {
```

Comment:
both are exactly same function, why don't we reuse the first one with better name?